packages/vsce/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the "cics-extension-for-zowe" extension will be documented in this file.
 
+## Recent Changes
+
+- Remove show attributes from tree menu [#422](https://github.com/zowe/cics-for-zowe-client/issues/422)
```

Comment:
I believe we should have a change-log that explains that the "Show Attributes" option is renamed to "Inspect Resource" as this most accurately describes to the user the change. Bare in mind users have not yet seen Inspect Resource in the context menu. Something like...

"Replaced the Show Attributes option with Inspect Resource in resource context menu, leveraging the updated CICS Resource Inspector."